tests/pytests/test_numbers.py

```diff
@@ -389,3 +389,106 @@ def testNegativeValues(env):
     # Query the index. if the split value of the root is nan, the query won't return any results.
     res = env.cmd('FT.SEARCH', 'idx', '@num:[-inf +inf]', 'NOCONTENT')
     env.assertEqual(res[0], doc_id)
+
+def test_numeric_operators(env):
+    env = Env(moduleArgs = 'DEFAULT_DIALECT 2')
+    conn = getConnectionByEnv(env)
+
+    env.expect('FT.CREATE', 'idx', 'SCHEMA', 'n', 'NUMERIC').ok()
+    waitForIndex(env, 'idx')
+
+    env.assertEqual(conn.execute_command('HSET', 'key1', 'n', '11'), 1)
+    env.assertEqual(conn.execute_command('HSET', 'key2', 'n', '12'), 1)
+    env.assertEqual(conn.execute_command('HSET', 'key3', 'n', '13'), 1)
+    env.assertEqual(conn.execute_command('HSET', 'key4', 'n', '14'), 1)
+    env.assertEqual(conn.execute_command('HSET', 'key5', 'n', '15'), 1)
+    env.assertEqual(conn.execute_command('HSET', 'key6', 'n', '-10'), 1)
+    env.assertEqual(conn.execute_command('HSET', 'key7', 'n', '-11'), 1)
+
+    # Test >= and <=
+    res1 = env.cmd('FT.SEARCH', 'idx', '@n:>=12 @n:<=14', 'NOCONTENT')
+    env.assertEqual(res1, [3, 'key2', 'key3', 'key4'])
+    res2 = env.cmd('FT.SEARCH', 'idx', '@n:>= + 12 @n:<=+  14', 'NOCONTENT')
+    env.assertEqual(res2, res1)
+    res2 = env.cmd('FT.SEARCH', 'idx', '@n:>=$min @n:<=$max',
```

Comment:
OK. Tests included. [e0e0b4d](https://github.com/RediSearch/RediSearch/pull/4516/commits/e0e0b4dd884dc45e824c991353abedf97b1e579c)
Tests for `-$param` were not included because it is not supported until now

Reply:
Will be added in #4505